modules/applyMiddleware.js

```diff
@@ -1,29 +1,23 @@
-import React, { cloneElement } from 'react'
+import React, { createElement } from 'react'
 import RouterContext from 'react-router/lib/RouterContext'
 
 export default (...middleware) => (
-  ((createElement) => (
+  ((makeCreateElement) => ((renderProps) => (
     middleware.filter(m => m.renderRootContainer).reduceRight(
```

Comment:
Probably best not to filter this every time. It gets messier without using variables though 😛